index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey microverse website</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <h1 class="heading-primary">Microverse Survey Process</h1>
+      </header>
+      <div id="container">
+        <section class="form-wrap">
+          <h2>Survey</h2>
+          <p>It's free and only takes a minute</p>
+          <form>
+            <div class="form-group">
+              <label for="name">Name</label>
+              <input type="text" id="name" placeholder="Enter your name" />
+            </div>
+            <div class="form-group">
+              <label for="sex">Sex</label>
+              <select name="sex" id="sex">
+                <option value="male" selected>Male</option>
+                <option value="female">Female</option>
+                <option value="other">Other</option>
+              </select>
+            </div>
+            <div class="form-group">
+              <label for="email">Email</label>
+              <input type="email" id="email" placeholder="Enter your email" />
+            </div>
+
+            <div class="form-group">
+              <label for="birthdate">Birthdate</label>
+              <input type="date" name="birthdate" id="birthdate" />
+            </div>
+            <div class="form-group">
+              <label for="quetion">Where do you heard about Microverse</label>
+              <select name="quetion" id="quetion">
+                <option value="facebook" selected>Facebook</option>
+                <option value="google">Google Search</option>
+                <option value="friend">Friend</option>
+              </select>
+            </div>
+
+            <div class="form-control">
+              <label>What your Programming level skill:</label>
+              <input type="radio" name="membership" value="simple" />
+              Basic
+              <input type="radio" name="membership" value="standard" checked />
+              Intermediate
+              <input type="radio" name="membership" value="super" />
+              Advance
+            </div>
```

Comment:
* Like the previous code reviewer pointed you still have not used the Github inline comment feature
* This is an example inline comment. 🙂

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.